understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
     <section id="brief">
 			<h2>In brief</h2>
 			<dl>
-				<dt>Goal</dt><dd>Content can be enlarged without increasing line length.</dd>
-				<dt>What to do</dt><dd>Make lines of text reflow within the viewport.</dd>
-        <dt>Why it's important</dt><dd>People who need bigger text find it difficult if they must scroll to read long lines.</dd>
+				<dt>Goal</dt><dd>Content can be enlarged without needing to scroll in two dimensions.</dd>
+				<dt>What to do</dt><dd>Make long lines of text and other applicable content reflow within resized or zoomed in interfaces.</dd>
```

Comment:
This far exceeds the length of any of the other In Brief material (which had a target of 10 words or less), with little clear benefit. Is the rationale for the change that people don't know what a viewport is? I'm also very concerned that "other applicable content" has been worked in here. We have clear support in discussions that text was the primary consideration.

Reply:
as mentioned on the call, "other applicable content" was in regards to images, buttons, other phrasing content that might not have been understood to fall under "text" per feedback from @patrickhlauke.

but we also said on the call we could just make it clear that "text / lines of text" might include other bits that are text-adjacent.  so agreed, we can simplify this down.